tests/wakunode_rest/test_rest_store.nim

```diff
@@ -775,7 +778,8 @@ procSuite "Waku Rest API - Store v3":
 
     var wakuMessages = newSeq[WakuMessage](0)
     for j in 0 ..< response.data.messages.len:
-      wakuMessages.add(response.data.messages[j].message)
+      if response.data.messages[j].message.isSome():
+        wakuMessages.add(response.data.messages[j].message.get())
```

Comment:
Unrelated to the actual change but maybe we could use `filterIt` in this case.

Something like:

`let wakuMessages = response.data.messages.filterIt(it.isSome())`